tests/unit/test_charm.py

```diff
@@ -218,6 +219,55 @@ def test_given_invalid_gnbsubnet_config_when_config_changed_then_status_is_block
             "The following configurations are not valid: ['gnb-subnet']"
         )
 
+    def test_given_gnbsubnet_config_is_not_cidr_when_config_changed_then_status_is_blocked(self):
```

Comment:
would it be worth it to make these tests parameterized?

Reply:
Sure, good idea.
Done.